starter-code/pizza.js

```diff
@@ -1 +1,71 @@
 // Write your Pizza Builder JavaScript in this file.
+$(document).ready(function() {
+var total = 13;
+//Remove the white sauce and the gluten free from the default order.
+$(".crust").toggleClass("crust-gluten-free");
+$(".sauce").toggleClass("sauce-white");
+
+//Remove the active class from the button Sauce and Crust
+$('.btn-sauce').removeClass("active");
+$('.btn-crust').removeClass("active");
+//Remove element that should not come y default in the default order.
+$(".price li:nth-child(4)").toggle();
+$(".price li:nth-child(5)").toggle();
+//Update Correctly the total price
+$(".price strong").html("$"+total);
+
+//on Click function of the button Sauce
+$('.btn-sauce').on("click", function(){
+  $(".sauce").toggleClass("sauce-white");
+  $('.btn-sauce').toggleClass("active");
+  $(".price li:nth-child(4)").toggle();
+  //Call to the function calculatePrice to update the price with the cost of the ingredient (first parameter).
+  //The second parameter is the element that needs to be check if its activated or not to modify the price correctly.
+  calculatePrice(3, ".price li:nth-child(4)");
+});
+
```

Comment:
the following click event listeners are pretty similar between each other, you could refactor everything in one function and handle the differences by passing parameters to the function